integrations/salesforce/src/handlers/handleProxyRequest.ts

```diff
@@ -0,0 +1,46 @@
+import axios from "axios";
+import { HandlerProps } from "src/misc/types";
+import { getSfCredentials, verifyPat } from "src/misc/utils/bpUtils";
+import { refreshToken } from "src/misc/utils/sfUtils";
+
+export const handleProxyRequest = async ({
```

Comment:
just saw it was commented out, maybe we can remove this code.

Reply:
Agree. REmoved